package.json

```diff
@@ -32,10 +32,8 @@
   "devDependencies": {
     "babel-cli": "^6.3.17",
     "babel-core": "^6.3.17",
-    "babel-loader": "^6.2.0",
-    "babel-plugin-add-module-exports": "^0.1.1",
-    "babel-plugin-syntax-function-bind": "^6.8.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.3.16",
```

Comment:
Babel only uses the transforms and plugins you specify in `.babelrc` and the ones that were on lines 37 and 38 were not being included there so they were cruft